src/Pages/sanghomoon/Main/MainLeft/Feeds.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import CommentList from "./CommentList";
+import "./Feeds.scss";
+
+class Feeds extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: "",
+      commentList: [],
+    };
+  }
+
+  index = 0;
+
+  getCommentValue = (e) => {
+    const { value } = e.target;
+    this.setState({
+      commentValue: value,
+    });
+  };
+
+  postComment = () => {
+    const { commentValue, commentList } = this.state;
+    this.setState({
+      commentList: commentList.concat({
+        index: this.index,
+        comment: commentValue,
+      }),
```

Comment:
concat은 정확하게는 배열과 배열을 합쳐주는 메서드라서 배열로 한 번 감싸주셔야 합니다!
```suggestion
      commentList: commentList.concat([{
        index: this.index,
        comment: commentValue,
      }]),
```

Reply:
네 다음에 반영해서 커밋 하겠습니다!